lib/kwtool_web/templates/layout/landing.html.eex

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>KWTool - Keyword Research Tool</title>
+    <link rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
+    <script defer type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
+  </head>
+  <body class="<%= page_class_name(@conn) %>">
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
+      <a class="navbar-brand" href="/"><i class="fas fa-binoculars"></i> KWTool</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarDefault"
+              aria-controls="navbarDefault" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbarDefault">
+        <ul class="navbar-nav mr-auto">
+          <%= if @conn.assigns.current_user == nil do %>
+            <li class="nav-item">
+              <%= link "Log in", class: "nav-link", to: Routes.session_path(@conn, :new) %>
+            </li>
+            <li class="nav-item">
+              <%= link "Sign up", class: "nav-link", to: Routes.auth_path(@conn, :show) %>
+            </li>
+          <% else %>
+            <li class="nav-item">
+              <%= link "Dashboard", class: "nav-link", to: Routes.home_path(@conn, :index) %>
+            </li>
+            <li class="nav-item">
+              <%= link "Logout", class: "nav-link", to: Routes.session_path(@conn, :delete), method: :delete %>
+            </li>
+          <% end %>
+        </ul>
+      </div>
```

Comment:
Later maybe we can split this navbar to a partial view as it is using on both app and landing layout 💡

Reply:
yes, that's good idea, i moved into partials view to make it more reusable.